schema/pool.go

```diff
@@ -0,0 +1,19 @@
+package schema
+
+import "github.com/gofrs/uuid"
+
+type (
+	JsonSchema struct {
+		ID  uuid.UUID `json:"-" faker:"uuid" db:"id" rw:"r"`
+		Url string    `json:"-" faker:"-" db:"url"`
+	}
+
+	Pool interface {
```

Comment:
This should be renamed to `Persister`. We use `Pool` only in identity and it's something I wanted to rename for some while...